### DISCUSSION TITLE
Broken link

### DISCUSSION THREAD

**Discussion by mdcll** [abraunegg/onedrive#2905] [Q&A] [answered]
I have been an user of linux client for onedrive. I started with "skillion" and after with "abraunegg/onedrive". Now I can't use because the unit where the "onedrive" is It have the link broken. Now I have to start w10 to update the files of onedrive. I use LMDE 6. I tray to fix it reinstaaling but the problem still. 
I have found something in google  but without solutions

[https://www.reddit.com/r/linuxquestions/comments/193wqe5/win10_onedrive_folder_appearing_as_broken_link_on/?rdt=46521](https://www.reddit.com/r/linuxquestions/comments/193wqe5/win10_onedrive_folder_appearing_as_broken_link_on/?rdt=46521)
[https://aur.archlinux.org/packages/ntfs-3g-onedrive-bin](https://aur.archlinux.org/packages/ntfs-3g-onedrive-bin)

Can you help me?

**Comment by abraunegg** [comment]
@mdcll 
> Now I can't use because the unit where the "onedrive" is It have the link broken

What do you mean by this ? What link is broken ?

**Comment by mdcll** [comment]
Thanks for your speed answer.
All the files of onedrive are in a unit called Onedrive.  It is a NTFS file system. I can see all de others files and directories but not the directory of onedrive It have a broken link (you can see the image). I suposse that it is not a problem of the linux onedrive client. I think it is a problem between Onedrive (in w10) and Linux but i don't know what to do to fix it.
![Screenshot from 2024-10-14 14-04-11](https://github.com/user-attachments/assets/d4547446-880a-4e68-a966-4bde907b88c6)

**Comment by abraunegg** [comment, marked as answer]
@mdcll 
> I suposse that it is not a problem of the linux onedrive client. I think it is a problem between Onedrive (in w10) and Linux but i don't know what to do to fix it.

There are a number of issues you could be facing:

1. Windows 'on-demand' functionality that is problematic for Linux
2. NTFS / BitLocker
3. Need to use other utilities to 'read|write' the Windows partition

You have a number of options:
1. Dont use 'on-demand' and sync your data to a separate drive / directory formatted with FAT32 which will then work with this client
2. Use https://github.com/gbrielgustavo/ntfs-3g-onedrive/ to read|write your Windows partition. Please read https://github.com/abraunegg/onedrive/discussions/2531 for details - but depends on you also most likely disabling 'on-demand' for Windows. This 'version' of ntfs-3g has a very specific update to read the 'on-demand' files that Windows OneDrive creates.
3. Use 'on-demand' and https://github.com/jstaf/onedriver

**Reply by mdcll** [reply]
Thanks again for your answer.

I have some problems with the 2n solution: (https://github.com/gbrielgustavo/ntfs-3g-onedrive/ )

![image](https://github.com/user-attachments/assets/57f5a7c2-a87f-4844-a457-4b06cba42bf4)
when install Fuse...remove ntfs-3g and others
```
manolo@08320:~/ntfs-3g-onedrive$ tail /var/log/apt/history.log 
Requested-By: manolo (1000)
Install: fuse:amd64 (2.9.9-6+b1)
Remove: ntfs-3g:amd64 (1:2022.10.3-1+b1), blueman:amd64 (2.3.5+mint1+faye), xdg-desktop-portal-gtk:amd64 (1.14.1-1), mintstick:amd64 (1.6.2), cinnamon-dbg:amd64 (6.2.9+faye), gvfs-fuse:amd64 (1.50.3-1), fuse3:amd64 (3.14.0-4), cinnamon:amd64 (6.2.9+faye), xdg-desktop-portal-xapp:amd64 (1.0.9+faye), flatpak:amd64 (1.14.10-1~deb12u1), mint-meta-cinnamon:amd64 (2023.09.08.1+lmde6), xdg-desktop-portal:amd64 (1.16.0-2)
End-Date: 2024-10-15  19:14:05
```
The fuse-libs does not exist in Debian 12.
when:
```
manolo@08320:~/ntfs-3g-onedrive$ libtoolize --force
aclocal
autoheader
automake --force-missing --add-missing
autoconf
./configure
```
It end with this message:
```
checking for libntfs-3g >= 2016.2.22AR... no
configure: error: "Unable to find libntfs-3g"

manolo@08320:~/ntfs-3g-onedrive$ aptitude search libntfs-3g
i   libntfs-3g89                                           - read/write NTFS driver for FUSE (runtime library)                
p   libntfs-3g89:i386                                      - read/write NTFS driver for FUSE (runtime library)                
manolo@08320:~/ntfs-3g-onedrive$
```
any suggestions?

**Reply by abraunegg** [reply]
Please seek support from https://github.com/gbrielgustavo/ntfs-3g-onedrive/ ... I cant assist you with something I did not write or use!

**Reply by mdcll** [reply]
Anyway, thanks a lot for your time and help.
